components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.html

```diff
@@ -1,17 +1,20 @@
-<div class="search-wrapper" tabindex="0">
-  <div class="search-input-wrapper">      
-    <input #search_box chefInput
-      id="search_box"
-      aria-label="Search"
-      (keyup)="handleInput($event.key, search_box.value)"
-      [value]="inputText"
-      type="text"
-      class="search-input"
-      placeholder="Search {{ placeHolder }}"
-      autocomplete="off">
-    <button chef-button class="search-button primary" primary (click)="handleFiltersClick(search_box.value)">
-      <chef-icon>search</chef-icon>
-    </button>
-  </div>
-  <ng-content></ng-content>
-</div>
+<form role='search'
+  (ngSubmit)="onSubmit(search_box.value)"
+  [class.active]="formActive">
+  <label for="search-filter" aria-hidden="true" class="hidden">
```

Comment:
Author note: For accessibility, all inputs should have a label, even if now viewable on the screen so that screen readers may access them.